src/richie/apps/courses/templatetags/extra_tags.py

```diff
@@ -23,7 +23,9 @@
 from cms.utils import get_site_id
 from cms.utils.plugins import get_plugins
 
+from .. import enums
```

Comment:
If I'm not mistaken this file does what we usually place in a  `defaults.py` file